### ISSUE TITLE
F80 code example doesn't fail

### ISSUE THREAD

**Issue by kengdoj** [w3c/wcag#3558]
https://www.w3.org/WAI/WCAG22/Techniques/failures/F80

At 400% zoom with Chrome and Edge, everything including the input field text increased.

**Comment by JAWS-test** [comment]
This is only true if the browser zoom is changed directly. However, if a different font size is selected in the browser, the inputs are not displayed enlarged

**Comment by JAWS-test** [comment]
However, it would be even better if it were used `input {font-size: 1vw;}` instead, because then the input would not scale even with browser zoom

**Comment by patrickhlauke** [comment]
> This is only true if the browser zoom is changed directly. 

and since 1.4.4 doesn't require a specific approach, but only that there be *a* way to enlarge text, this technique passes. agree that it should either be changed or removed /cc @alastc

**Comment by bruce-usab** [comment]
Any other ideas for failure techniques illustrating how author might break browser zoom?

Thanks @JAWS-test for `1vw` as simple substitution!  I tried 12px (instead of 12pt) but browser zoom worked fine.

**Comment by Jym77** [comment]
Same problem with the example in F69, they work perfectly fine with browser zoom and only break with font size change (`vw` font size doesn't do the trick here since it makes the text overflow on small zoom and fit on big zoom).

**Comment by Jym77** [comment]
Looking into that again, it seems that even [F94](https://www.w3.org/WAI/WCAG22/Techniques/failures/F94) may need a rework.

When I'm looking at [the detailed example](https://www.w3.org/WAI/WCAG22/working-examples/failure-viewport-units/) in Firefox and use the "zoom text only" setting, it scales nicely. Other methods (usual browser zoom, increasing font size) do not let the text be resized, though.

Judging by [Understanding 1.4.4](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html), it seems that "zoom text only" is a valid way of passing it:
> a number of mechanisms - including zoom (of the entire page's content), magnification, **text-only resizing**, and allowing the user to configure a size for rendered text.

(emphasize mine)

And with the recent clarification that
> Content satisfies the Success Criterion if it can be scaled up to 200% using at least one text scaling mechanism supported by user agents.

That page seems to now satisfy 1.4.4.

**Comment by patrickhlauke** [comment]
> Judging by [Understanding 1.4.4](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html), it seems that "zoom text only" is a valid way of passing it:

however, the option to zoom text only is either hidden, or even completely absent, in most browsers these days, and full-page zoom is pretty much the default (or only) behavior. so it's mostly an edge case these days (and effectively, the browser is rewriting/ignoring the styles set by the author, in effect...which is already a case that is arguably outside of the author's direct control)

**Comment by patrickhlauke** [comment]
going right back to the original question ... I'd agree that F80 (and any other failure techniques relating to text resizing, that now don't fail anymore in the de-facto standard of full-page zoom as the default in most/all browsers) need to either be completely deleted or at least caveated with a big warning block saying that they're deprecated

**Comment by patrickhlauke** [comment]
From a tiny bit of testing (and making codepen working examples for those failure that didn't have them):

* https://www.w3.org/WAI/WCAG22/Techniques/failures/F69 / https://es-d-2867392120260428-019dcba7-b0a1-7da0-844b-d5ad648e370d.codepen.dev/ works fine with full-page zoom, fails in Firefox and Safari when doing text-only zoom
* https://www.w3.org/WAI/WCAG22/Techniques/failures/F80 / https://es-d-2867392120260428-019dcba1-9715-724d-9b37-30ac9abeb2a2.codepen.dev/ works fine with full-page zoom *and* in Firefox and Safari when doing text-only zoom
* https://www.w3.org/WAI/WCAG22/Techniques/failures/F94 / https://www.w3.org/WAI/WCAG22/working-examples/failure-viewport-units/ fails when using full-page zoom in Chrome and Firefox, but oddly doesn't fail in Safari 26 (there appears to be a bug in Safari, as `vh` should *not* behave that way); Firefox and Safari the `vh` set text *also* resizes (again, it shouldn't?)

Note the Firefox has half-hidden its option to do text-only zooming in Windows - it can only be reached when forcing the old menu bar to appear, then it's in View > Zoom - this menu bar is still prominently visible by default in macOS; otherwise, you have to go to Settings > General > Zoom. In Safari, the option appears to be completely hidden, and can only be triggered with the keyboard using Option+Command and then plus/minus (regular full-page zooming is just Command and then plus/minus). Chrome and Edge don't appear to have a text-only zoom option, so if content only works with *that* but *not* full-page zoom, it's not consistently available enough to count as a pass.

Long story short, I think this needs two things: a clarification in understanding that by far the most common method of resizing text is full-page zoom (as that's what browsers surface the most, and is consistent across browsers as an option). Then, the three failure techniques listed need to either be changed, or deprecated for now. And as bonus, Firefox and Safari's behaviour with `vh` seems non-standard and bugs should be filed.

**Comment by patrickhlauke** [comment]
bug filed for Safari/macOS https://bugs.webkit.org/show_bug.cgi?id=313495 / video recording https://www.youtube.com/watch?v=DzPUEVOAFJk

edit: my bug was marked as duplicate of https://bugs.webkit.org/show_bug.cgi?id=313456 which was filed earlier on the same day :)
